UM/Qt/Bindings/ContainerProxy.py

```diff
@@ -1,4 +1,5 @@
-from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
+from PyQt5.QtCore import QObject
+from UM.FlameProfiler import pyqtSlot
```

Comment:
If we forget it then nothing breaks. It will just be invisible. In some places we may want to remove the pyqtSlot stuff if it isn't producing anything interesting and just spamming up the results.